projects/adrv9009/src/app/headless.c

```diff
@@ -760,13 +772,30 @@ int main(void)
 	/* Initialize the ADC core */
 	axi_adc_init(&rx_adc, &rx_adc_init);
 
+#ifdef DAC_DMA_EXAMPLE
+	gpio_get(&gpio_plddrbypass, DAC_FIFO_BYPASS);
+	gpio_direction_output(gpio_plddrbypass, 1);
+	axi_dac_load_custom_data(tx_dac,
+			sine_lut_iq,
+			sizeof(sine_lut_iq) / sizeof(uint32_t),
+			XPAR_DDR_MEM_BASEADDR + 0xA000000);
```

Comment:
XPAR_DDR_MEM_BASEADDR is Xilinx dependent. DDR_MEM_BASEADDR should be used instead - it is defined in projects/adrv9009/src/devices/adi_hal/parameters.h .